src/components/CircularProgressBar/CircularProgressBar.js

```diff
@@ -57,6 +57,7 @@ const CircularProgressBar = ({
 
   useEffect(() => {
     Animated.timing(circleAnim, { toValue: progress, duration: 500, useNativeDriver: true }).start();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
If we don't like `react-hooks/exhaustive-deps` that much, can't we disable it globally?

Reply:
It's a recommended hook, which this PR enables. However we disable are existing rule violations, so that it will only apply to new code.